auth/configurations/jwt.ts

```diff
@@ -0,0 +1,8 @@
+let jwtConfig: any = {};
+
+jwtConfig.encryption =
+  process.env.JWT_ENCRYPTION ||
+  "`#{E)EZVefmnkswfoksegjiergjkskjgsgkjsngksgvnskvnskgvnsknknknknknksngvskvnsvkjsvjlksdjrjrngjrgnjrgnrjgnjR)F*2`d6";
```

Comment:
is it safe to put this here?

Reply:
This line assigns the encryption key to a random string if the environment variable JWT_ENCRYPTION wasn't assigned. In production, all environment variables (which also includes db details and so on) would be manually assigned.